crates/common/src/telemetry.rs

```diff
@@ -0,0 +1,75 @@
+use std::net::SocketAddr;
```

Comment:
I suggest separate crate

Reply:
Common is the crate which groups code that is shared between different node types. I don't think a separate crate is needed for this.